src/Hubbup.Web/Pages/Index.cshtml

```diff
@@ -0,0 +1,308 @@
+                    {{#if milestone}}
+                        {{> badge milestone.title}}
+                    {{else}}
+                        {{> badge "< No Milestone >"}}
+                    {{/if}}
+                {{/unless}}
+            </div>
+        </div>
+    </script>
+
+    <script>
+        var lastRestRateLimit = null;
+        var globalCost = 0;
+            
+        function loaded() {
+            applyTemplate(rateLimitDisplay, rateLimitTemplate, {
+                remaining: "?",
+                alertClass: "alert-info"
+            });
+            var panels = document.querySelectorAll(".i-assignee-panel")
+            panels.forEach(bindPanel);
+        }
+
+        function bindPanel(panel) {
+            var assignee = panel.dataset.assignee;
+            var id = panel.dataset.id;
+
+            // Start the initial load
+            setTimeout(() => {
+                refresh(assignee, id, panel);
+            }, 0);
+        }
+
+        function refresh(assignee, id, panel) {
+            var parameters = {
+                id: id,
+                assignee: assignee
+            };
+
+            applyTemplate(panel, assigneePanelTemplate, parameters);
+
+            // Fetch data
+            fetch(`/api/groups/@(Model.GroupName)/issues/${assignee}`, {
+                credentials: "same-origin"
+            }).then(response => {
+                return response.json();
+            }).then(json => {
+                parameters.rateLimit = json.graphQlRateLimit;
+                parameters.restRateLimit = json.restRateLimit;
+                parameters.pages = json.pages;
+                parameters.data = {
+                    working: json.working,
+                    prs: json.prs,
+                    other: json.other,
+                    id: id
+                };
+
+                globalCost += json.graphQlRateLimit.cost;
+
+                // Update rate limit display
+                if (json.restRateLimit) {
+                    lastRestRateLimit = json.restRateLimit
+                }
+                var rateLimitInfo = {
+                    globalCost: globalCost,
+                    graphql: getRateLimitInfo(json.graphQlRateLimit.remaining),
+                    rest: lastRestRateLimit && getRateLimitInfo(lastRestRateLimit.remaining)
+                }
+
+                applyTemplate(rateLimitDisplay, rateLimitTemplate, rateLimitInfo);
+
+                applyTemplate(panel, assigneePanelTemplate, parameters);
+            }).catch(err => {
+                parameters.errorMessage = err.message;
+
+                applyTemplate(panel, assigneePanelTemplate, parameters);
+            });
+        }
+
+        function getRateLimitInfo(remaining) {
+            var rateLimitInfo = {
+                remaining: remaining
+            };
+            if (remaining < 1000) {
+                rateLimitInfo.alertClass = "alert-danger";
+            }
+            else if(remaining < 2000) {
```

Comment:
These are a particular pain because I can't reformat because of https://github.com/aspnet/Razor/issues/1394